### ISSUE TITLE
Surname Tradition - Icelandic — Children take a patronym instead of a surname - Change

### ISSUE THREAD

**Issue by gudjonsi** [fisharebest/webtrees#4575]
I would like to suggest a change to the current setting (code) for the Icelandic surname tradition already established in webtrees. I recommend that the last name or surname of an Icelandic person be created in the surname field of webtrees instead of the given name field. The reason behind this is that the patronymic name establised is based on in general the fathers given names with the suffix addition -sson for sons and -sdóttir for daugthers and is not the persons given name but rather a last name (surname) indicating relationship to his/her father and in rare instances to his/her mother. Also if the last name is not inserted in the surname field of webtrees the statistic of all last names in an Icelandic family tree can not be displayed and consequently not providing information on the number of different last names found in a family tree.
Icelanders always go by their given names (or first names) and not their last names. In genealogy we would use both the given name GIVN and the surname SURN in GEDCOM to distinguish the two separate part of the Icelandic naming system, i.e. the first names (given names) and the last names (surname or relation names) of a person.
Icelandic surnames are different from most other naming systems in the modern Western world by being patronymic or occasionally matronymic: they indicate the father (or mother) of the child and not the historic family lineage. The Icelandic system is thus not based on family names (although some people do have family names and might use both systems). Generally, with few exceptions, a person’s last name indicates the first name of their father (patronymic) or in some cases mother (matronymic) in the genitive, followed by -son (“son”) or -dóttir (“daughter”). 
Please change the code to reflect this by establishing the patronymic last name (Icelandic naming tradition) in the surname field instead of the given name field and add appropriate slashes in the name field for the last name. Also change the last name suffix for daughters from -sdottir to -sdóttir, see enclosed image.
![Image 12 10 2022 at 19 54](https://user-images.githubusercontent.com/100283838/195586129-ae873696-2b9e-42d2-8e3b-3bc241042358.jpeg)
![Image 12 10 2022 at 19 55](https://user-images.githubusercontent.com/100283838/195586135-29b4a86e-5b79-4d6c-88b6-02abe1737b8b.jpeg)

**Comment by Norwegian-Sardines** [comment]
Norway has this issue as well!   Probably other Nordic and older naming customs as well!

**Comment by fisharebest** [comment]
I agree the current logic is incorrect - the patronym/matronym should not be part of the given-name field.

But it is not a surname, and therefore should not have slashes ???

Instead, I think it would be more correcct to store the name like this:

```
1 NAME Johan Olafsson
2 GIVN Johan
```

So, when we add a son to Olaf, we should create a form containing just Olafsson in the "full name" field, and leave all others empty...

**Comment by gudjonsi** [comment]
Ok, but if we don't use the surname field in webtrees for patronymic/matronymic last names they will most likely not be indexed in webtrees and displayed in the statistics for a family tree. It will not be possible to identify the most used patronymic/matronymic last names (surnames).

I would recommend to use the surname field in webtrees for both patronymic/matronymic and family names because both can occur in the Icelandic naming system, see information below from the University of Iceland:

```
Is people's last name part of their given name?

In the law on personal names no. 45 from 1996, the first chapter deals with full name and naming. Article 1 says: "A person's full name is his first name or first names, middle name, if there is one, and last name." If you look at the article about first names in the fourth chapter, it says in the fifth article:

A given name must be able to take the Icelandic genitive case or have a tradition in the Icelandic language. The name must not violate the Icelandic language system. It must be written in accordance with the general writing rules of the Icelandic language, unless there is a tradition for another way of writing it. A girl should be given a female name and a boy should be given a male name. A given name must not be such that it can cause trouble for the bearer of the name.

In the fourth chapter, the eighth article, identification names are discussed. It says: "Identification names are of two kinds, father's or mother's names and family names."

If you look up the Icelandic dictionary (2002:255) and look for the word last name, the explanation is: "surname (sometimes also identification name)", for identification name it says (2002:764): "father's or mother's name" but for given name (2002 :259): "a name by which a person is called (not a identification name or family name)".

From what has been summarized here, it can be seen that a last name is not part of a given name. A given name is the name that a child receives when naming, while the identification name (surname) comes from the mother, father or family.

Author: Guðrún Kvaran, professor emeritus.
The Science Web, University of Iceland. https://www.visindavefur.is/svar.php?id=74417#

```
In Iceland we can have the following multiple last names (surnames) for a single person (patronymic, matronymic and family name), although not very common but is authorised in the law for the Icelandic naming system:

Example: Ingibjörg Jóna Guðrúnardóttir Sveinsdóttir Gröndal
Definition: Given name, Given name, Matronymic last name, Patronymic last name, Family name

1 NAME Ingibjörg Jóna /Guðrúnardóttir Jónsdóttir Gröndal/
2 TYPE BIRTH
2 GIVN Ingibjörg Jóna
2 SURN Guðrúnardóttir Jónsdóttir Gröndal

and perhaps to index all last names and family names (surnames):

1 NAME Ingibjörg Jóna /Guðrúnardóttir/ /Jónsdóttir/ /Gröndal/
2 TYPE BIRTH
2 GIVN Ingibjörg Jóna
2 SURN Guðrúnardóttir, Jónsdóttir, Gröndal

I would strongly recommend to use the SURN field in GEDCOM for Icelandic Patronymic/Matronymic last names and family names to be able to separate the two different name parts of the Icelandic names in first names (GIVN) and last names (SURN)  (both patronymic/matronymic and family names). This would make it possible to index all last names for statistics and also to export and import GEDCOM to perhaps another genealogy application and keep the distinct separation between first names and last names in the Icelandic naming system.

Is the argument for excluding the use of SURN field in GEDCOM for patronymic/matronymic perhaps involving the applicable code in webtrees for generating patronymic/matronymic last names or does the GEDCOM standard 5.5.1 or newer standards exclude the use of patronymic/matronymic last names in the SURN field?

In FamilySearch it appears that both GIVN and SURN fields are being used for all Icelanders recorded in their database, SURN contains patronymic last names for both males and females.

**Comment by gudjonsi** [comment]
I did some testing with a family tree of five persons with different GEDCOM recording as follows. The family tree is set to Icelandic patronymic naming system.

1 NAME Jón /Sigurðsson/
2 GIVN Jón
2 SURN Sigurðsson

1 NAME Anna Jónsdóttir Gröndal
2 TYPE BIRTH
2 GIVN Anna
2 SURN Gröndal

1 NAME Ingibjörg Jóna Önnudóttir Jónsdóttir Gröndal
2 TYPE BIRTH
2 GIVN Ingibjörg Jóna
2 SURN Önnudóttir Jónsdóttir Gröndal

1 NAME Sveinn Jónsson
2 TYPE BIRTH
2 GIVN Sveinn

1 NAME Bylgja Jónsdóttir
2 TYPE BIRTH
2 GIVN Bylgja

1. Sveinn Jónsson and Bylgja Jónsdóttir are not listed in the individual list (SURN tag not used and no slashes in NAME tag).
2. Only three individuals are listed in the individual list, those with their last name in the SURN tag, of family of five persons.
3. If the individuals with their last name in SURN are not with their last name in slashes in the NAME tag then their last names are not displayed in the individual list when filtered as surname but displayed as "No surname".
4. Same applies to the last names (surnames) listed in statistics, table and charts.

See enclosed images for verification.

![Image-1](https://user-images.githubusercontent.com/100283838/196812525-07efd373-7c6f-4c8a-ab87-1ae6a1c8b15d.jpeg)
![Image-2](https://user-images.githubusercontent.com/100283838/196812529-947d79ad-2995-469f-91fc-ee9d65da368a.jpeg)
![Image-3](https://user-images.githubusercontent.com/100283838/196812533-4963d0b6-ebdf-4d63-a73d-1236129daf93.jpeg)
![Image-4](https://user-images.githubusercontent.com/100283838/196812538-b1ffc641-2c3a-412e-be84-d6723e1c6da9.jpeg)
![Image-5](https://user-images.githubusercontent.com/100283838/196812540-7402abe0-4e82-478c-a344-33aa076487b1.jpeg)
![Image-6](https://user-images.githubusercontent.com/100283838/196812542-f5311368-7278-4b15-8c54-cc3357c45502.jpeg)
![Image-7](https://user-images.githubusercontent.com/100283838/196812544-40de223d-2ed2-4558-81b2-5686c5becbf9.jpeg)
![Image-8](https://user-images.githubusercontent.com/100283838/196812545-09b7d31a-c99b-4a2a-b4b2-66a1d57a3313.jpeg)
![Image-9](https://user-images.githubusercontent.com/100283838/196812546-b03324b5-2bb7-495f-a583-a256080b04fa.jpeg)

**Comment by Norwegian-Sardines** [comment]
> Is the argument for excluding the use of SURN field in GEDCOM for patronymic/matronymic perhaps involving the applicable code in webtrees for generating patronymic/matronymic last names or does the GEDCOM standard 5.5.1 or newer standards exclude the use of patronymic/matronymic last names in the SURN field?

One of the issue that has been raised with GEDCOM is that the NAME subtags are optional and that various programs import the information differently.  Some drop the subtags always, some use the subtags and ignore the NAME tag. None, to my knowledge, use the subtags for indexing separately from the NAME tag, except webtrees.

I have these problems in Norway as well, GEDCOM does not support patronymic names, nor do most non-Norwegians understand that historical patronymic names are not surnames.

These issues continue into GEDCOM v7.0 but may get some resolution in a future release.  Many culture also use place names as a way to give a person an identity.
